examples/tornjak-keycloak-client-import.json

```diff
@@ -0,0 +1,93 @@
+{
+      
+      "clientId": "Tornjak-React-auth",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "alwaysDisplayInConsole": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "redirectUris": [
+        "*"
+      ],
+      "webOrigins": [*],
```

Comment:
i don't know why an export caused this, but the star needs quotes or it will not be imported properly

Reply:
done